cryotypes/projectionmodel/_data_labels.py

```diff
@@ -0,0 +1,15 @@
+class ProjectionModelDataLabels:
+    ROTATION_X = "rotation_x"
+    ROTATION_Y = "rotation_y"
+    ROTATION_Z = "rotation_z"
+    ROTATION = [ROTATION_X, ROTATION_Y, ROTATION_Z]
```

Comment:
Should we maybe for consistency use actual `Rotation` objects internally?

Reply:
agree we want consistency but don't think a Rotation is simpler for this use case where a primary use case could be checking how much things have shifted from expected stage tilt angles -> I don't want to force understanding rotation decomposition on someone in that situation

Reply:
We can always provide utility functions... I mean, this still requires all the convention stuff from euler angles, so at least the Rotation moves the burden to the creation of the object, instead of both creation and utilisation. I just think consistency here would be nice to keep; and like with poses, it's just more convenient to work with single values rather than multiple things.

Either way, we should at least clarify in which order rotations happen, if they are intrinsic/extrinsicm, yada yada.

Reply:
note added explaining rotations!